recipe-server/client/control/tests/state/session/test_actions.js

```diff
@@ -0,0 +1,60 @@
+import { Map } from 'immutable';
+
+import {
+  SESSION_INFO_HISTORY_VIEW,
+} from 'control/state/action-types';
+
+import { addSessionView } from 'control/state/app/session/actions';
+
+describe('Session actions', () => {
+  describe('addSessionView', () => {
+    // Params = the 'default' test params passed into addSessionView
+    const defaultParams = ['category', 'caption', 'identicon'];
+    // Values = the 'default' dispatched values given our default params.
+    const defaultValues = { caption: 'caption', category: 'category', identicon: 'identicon' };
+
+
+    it('should dispatch a SESSION_INFO_HISTORY_VIEW event', async () => {
+      const meta = {
+        dispatch: () => {},
+      };
+      spyOn(meta, 'dispatch').and.callThrough();
```

Comment:
You can probably do this more cleanly with [`jasmine.createSpy`](https://jasmine.github.io/2.0/introduction.html#section-Spies:_%3Ccode%3EcreateSpy%3C/code%3E):